views/js/e2e/runner/_setup/setupCommands.js

```diff
@@ -19,6 +19,7 @@
 import runnerUrls from '../_urls/runnerUrls';
 import setupSelectors from './setupSelectors';
 import base64Test from './base64QtiExampleTestPackage';
+import runnerSelectors from "../_selectors/runnerSelectors";
```

Comment:
no runnerselectors for setupcommands, please extend setupSelectors

Reply:
yes